### PR TITLE
Blog app - views

### PR DESCRIPTION
**Pull request by bilalrajput09**
### Implemented the design from the sneak peek wireframes.

- Added all the elements needed for routing between different views.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,9 @@
-<h1>You are in show page of users_view</h1>
+<%= render @user %>
+<p>
+    <strong class="bio">Details: </strong>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Repellat odio harum reiciendis magnam, architecto voluptates ex doloremque ea optio quibusdam rem repudiandae est, fuga mollitia at sequi repellendus illum vitae. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ratione libero consequuntur nobis natus quia suscipit magni, dolores molestiae iusto reprehenderit repellat saepe itaque, quam sequi voluptate fugit distinctio minima aspernatur?
+</p>
+<%= render @posts %>
```
Comment (Davidosky007) [inline app/views/users/show.html.erb]: * As shown in the wireframe provided you are to render three of the users most recent posts you can write something like this ``<% @user.recent_post.each do |post| %>``
image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,2 @@
-<h1>You are in index page of posts_index</h1>
+<%= render @user%>
```
Comment (Davidosky007) [inline app/views/posts/index.html.erb:1]: As shown in the wireframe provided you are to use `post.recent_comments` to render 5 most recent comments you can write something like this 
````
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
````
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @bilalrajput09 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (bilalrajput09) [reply app/views/posts/index.html.erb:1]: Fixed

File: app/views/users/show.html.erb (hunk 1 above)
Comment (bilalrajput09) [reply app/views/users/show.html.erb]: Fixed

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @bilalrajput09 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested changes well implemented :+1: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**